package.json

```diff
@@ -1,8 +1,13 @@
 {
-  "name": "morpho-blue",
+  "name": "@morpho-org/morpho-blue",
   "description": "Morpho Blue Protocol",
   "license": "BUSL-1.1",
-  "version": "1.0.0",
+  "version": "v1.0.0",
```

Comment:
Resources:
- https://docs.npmjs.com/cli/v10/configuring-npm/package-json#version
- https://github.com/npm/node-semver

It seems this version **might** work. Why aren't we using `1.0.0` instead?

Reply:
Is it the syntax of `yarn version`??

Reply:
Yes, this is the default syntax used by version. Here we are using hardcoded version, but I've reused the default behavior of --patch, --minor & --major

Reply:
If I do it myself, I can increment versions using `yarn version --patch/minor/major` and it will keep the syntax `x.x.x` instead of `vx.x.x`